src/components/Navbar/index.jsx

```diff
@@ -70,20 +82,20 @@ export default function Navbar() {
               href='/donation'
               className='relative mx-4 capitalize text-sm hover:text-violet-800   transition-all duration-1000 before:absolute before:-bottom-2 before:left-0  before:h-[1px]  before:w-0 before:opacity-0 before:bg-violet-700 before:transition-all before:duration-500 hover:before:w-full hover:before:opacity-100'
```

Comment:
Why not cleaning the code and extracting these classes into a variable and use this variable instead of repeating the classes here and down this one??

Reply:
Well, it's better to create an issue to clean the code from all components.